superset/assets/javascripts/modules/AnnotationTypes.js

```diff
@@ -0,0 +1,29 @@
+import { VIZ_TYPES } from '../../visualizations/main';
+
+const ANNOTATION_TYPES = {
+  FORMULA: 'FORMULA',
+  EVENT: 'EVENT',
+  INTERVAL: 'INTERVAL',
+  TIME_SERIES: 'TIME_SERIES',
+  NATIVE: 'NATIVE',
```

Comment:
I think we should get rid of `NATIVE` here, and as discussed offline move to a model where you only have formulas, events, intervals, and time series. the distinction between what is currently called "native" and "interval" should be at the `source` level, not confounded with the `type`.